### PR TITLE
update pending 3.8

### PR DESCRIPTION
**Pull request by kathy-nevola**
# Title: Write a very short summary here
**Problem:** Write a description of your problem here.

**Solution:** Write a short summary of your solution.

**Key Features:**

* Key feature 1
* Key feature 2
* ...

## Checklist
- [ ] Make sure you are requesting to **pull a feature/bugfix branch** (right side). This should not be main or develop.
- [ ] Make sure you are make a pull request against either **main or develop** (left side). (Requesting to main should be reserved for bugfixes and new releases)
- [ ] Add or update unit tests (if applicable)
- [ ] Check your code with any unit tests (Run devtools::check() locally)
- [ ] Add neccessary documentation (if applicable)

## Type of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue) (link the issue on the right)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Documentation Update 
- [ ] Other (explain)

## Further comments
If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, any questions you have, etc...

Consider linking any issues (#issue-number ) or adding @mentions to ensure specific people see it.

### CODE REVIEW

File: OlinkAnalyze/NEWS.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# Olink Analyze 3.8.0
+## Minor Changes
+* Function support for calculating LOD for Explore data (#355, @kathy-nevola, @klevdiamanti, @leiliuC)
+* Tutorial and documentaiton for calculating LOD for Explore data (#364, @MasoumehSheikh, @kristynchin-olink)
```
Comment (kathy-nevola) [inline OlinkAnalyze/NEWS.md:4]:
```suggestion
* Tutorial and documentation for calculating LOD for Explore data (#364, @MasoumehSheikh, @kristynchin-olink)
```

File: OlinkAnalyze/DESCRIPTION [hunk 2]
```diff
@@ -41,8 +41,10 @@ Authors@R: c(
            comment = c(Github = "olofmansson")),
     person("Ola", "Caster", role = "ctb",
            comment = c(Github = "OlaCaster")),
-    person("Kristyn", "Chin", role = "ctb",
+    person("Kristyn", "Chin", role = "aut",
```
Comment (kristynchin-olink) [inline OlinkAnalyze/DESCRIPTION:44]: Only potential change - moving up below other authors for consistency. Not required though.

**Review by kristynchin-olink — APPROVED** [review]
Only potential change is moving all authors into same section for consistency. However this is not required and all changes look good.

**Review by kristynchin-olink — APPROVED** [review]
Authors moved together. All changes look good.

**Pull request merged**